### PR TITLE
update ntt faq

### PR DESCRIPTION
**Pull request by ilariae**
### Description

Updated NTT FAQ 

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

**Review by dawnkelly09 — DISMISSED** [review]
LGTM!

File: build/contract-integrations/native-token-transfers/faqs.md [hunk 1]
```diff
@@ -84,4 +84,37 @@ These functions ensure Connect can handle token transfers and manage chain-rate
 
 ## How does the relayer contract determine which transceiver to call?
 
-The source chain's transceiver includes the destination chain's transceiver in the message via the relayer contract. The admin configures each transceiver's mapping of its peers on other chains. This mapping allows the destination transceiver to verify that the message came from a trusted source.
\ No newline at end of file
+The source chain's transceiver includes the destination chain's transceiver in the message via the relayer contract. The admin configures each transceiver's mapping of its peers on other chains. This mapping allows the destination transceiver to verify that the message came from a trusted source.
+
+<!-- new questions from here -->
```
Comment (eshaben) [inline build/contract-integrations/native-token-transfers/faqs.md:89]: Just quickly looked at this. Does this need to be here?

**Review by eshaben — COMMENTED** [review]
Just one question 🙂

**Review by dawnkelly09 — APPROVED** [review]
lgtm!

**Pull request merged**